lib/config.js

```diff
@@ -9,6 +9,11 @@ const caPath = '/var/run/secrets/kubernetes.io/serviceaccount/ca.crt';
 const tokenPath = '/var/run/secrets/kubernetes.io/serviceaccount/token';
 const namespacePath = '/var/run/secrets/kubernetes.io/serviceaccount/namespace';
 
+function defaultConfigPath() {
```

Comment:
I couldn't decide whether you prefer to have `homeDir` and `defaultConfig` as constants or as a function as it is now.